JWBestPracticeApps/GoogleDAI/ViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ViewController.swift
+//  GoogleDAI
+//
+//  Created by Stephen Seibert  on 3/30/20.
+//  Copyright © 2020 Karim Mourra. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
@s-seibert you're not following the pattern in the best practice apps. The pattern is to inherent from View Controllers in simpler targets, that way you only have to show the logic needed to achieve a  specific goal.